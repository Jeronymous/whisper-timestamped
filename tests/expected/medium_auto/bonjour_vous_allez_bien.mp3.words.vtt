WEBVTT

00:00.440 --> 00:01.660
Bonjour!

00:01.660 --> 00:02.180
Est-ce

00:02.180 --> 00:02.280
que

00:02.280 --> 00:02.380
vous

00:02.380 --> 00:02.560
allez

00:02.560 --> 00:03.320
bien?

00:32.600 --> 00:34.240
Bonjour!

00:34.240 --> 00:34.720
Est-ce

00:34.720 --> 00:34.820
que

00:34.820 --> 00:34.920
vous

00:34.920 --> 00:35.080
allez

00:35.080 --> 00:35.480
bien?

