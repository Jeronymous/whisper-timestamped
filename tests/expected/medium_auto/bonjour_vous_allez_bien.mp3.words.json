{
  "text": " Bonjour ! Est-ce que vous allez bien ? Bonjour ! Est-ce que vous allez bien ?",
  "segments": [
    {
      "id": 0,
      "seek": 0,
      "start": 0.42,
      "end": 3.46,
      "text": " Bonjour ! Est-ce que vous allez bien ?",
      "tokens": [
        50364,
        25431,
        2298,
        4410,
        12,
        384,
        631,
        2630,
        18146,
        3610,
        2506,
        50564
      ],
      "temperature": 0.0,
      "avg_logprob": -0.3616434610806979,
      "compression_ratio": 0.8260869565217391,
      "no_speech_prob": 0.07169149070978165,
      "confidence": 0.936,
      "words": [
        {
          "text": "Bonjour !",
          "start": 0.42,
          "end": 1.92,
          "confidence": 0.874
        },
        {
          "text": "Est-ce",
          "start": 1.92,
          "end": 2.16,
          "confidence": 0.885
        },
        {
          "text": "que",
          "start": 2.16,
          "end": 2.24,
          "confidence": 0.988
        },
        {
          "text": "vous",
          "start": 2.24,
          "end": 2.38,
          "confidence": 0.996
        },
        {
          "text": "allez",
          "start": 2.38,
          "end": 2.58,
          "confidence": 0.99
        },
        {
          "text": "bien ?",
          "start": 2.58,
          "end": 3.46,
          "confidence": 0.999
        }
      ]
    },
    {
      "id": 1,
      "seek": 3000,
      "start": 32.94,
      "end": 35.86,
      "text": " Bonjour ! Est-ce que vous allez bien ?",
      "tokens": [
        50364,
        25431,
        2298,
        4410,
        12,
        384,
        631,
        2630,
        18146,
        3610,
        2506,
        50664
      ],
      "temperature": 0.0,
      "avg_logprob": -0.30651259422302246,
      "compression_ratio": 0.8260869565217391,
      "no_speech_prob": 0.40451279282569885,
      "confidence": 0.933,
      "words": [
        {
          "text": "Bonjour !",
          "start": 32.94,
          "end": 34.44,
          "confidence": 0.741
        },
        {
          "text": "Est-ce",
          "start": 34.44,
          "end": 34.7,
          "confidence": 0.921
        },
        {
          "text": "que",
          "start": 34.7,
          "end": 34.76,
          "confidence": 0.996
        },
        {
          "text": "vous",
          "start": 34.76,
          "end": 34.9,
          "confidence": 0.998
        },
        {
          "text": "allez",
          "start": 34.9,
          "end": 35.1,
          "confidence": 0.997
        },
        {
          "text": "bien ?",
          "start": 35.1,
          "end": 35.86,
          "confidence": 0.999
        }
      ]
    }
  ],
  "language": "fr",
  "language_probs": {
    "en": 0.06090574339032173,
    "zh": 0.003715273691341281,
    "de": 0.005754328332841396,
    "es": 0.007161377463489771,
    "ru": 0.0019886428490281105,
    "ko": 0.0026760115288197994,
    "fr": 0.7536683082580566,
    "ja": 0.002848599338904023,
    "pt": 0.0049219378270208836,
    "tr": 0.0009844496380537748,
    "pl": 0.0036009668838232756,
    "ca": 2.1080166334286332e-05,
    "nl": 0.002848599338904023,
    "ar": 0.00263452366925776,
    "sv": 0.00025680899852886796,
    "it": 0.0036576741840690374,
    "id": 0.00012321829854045063,
    "hi": 0.00041684217285364866,
    "fi": 8.468662417726591e-05,
    "vi": 0.0002489078033249825,
    "he": 4.325369809521362e-05,
    "uk": 4.0003127651289105e-05,
    "el": 7.591259782202542e-05,
    "ms": 0.0007202387205325067,
    "cs": 0.0002955853415187448,
    "ro": 9.596251766197383e-05,
    "da": 4.2583109461702406e-05,
    "hu": 7.357701542787254e-05,
    "ta": 7.634730536665302e-06,
    "no": 5.383009556680918e-05,
    "th": 0.00013745995238423347,
    "ur": 7.243630534503609e-05,
    "hr": 2.8528902475954965e-06,
    "bg": 1.1824902685475536e-05,
    "lt": 5.035650474383147e-07,
    "la": 0.02311721071600914,
    "mi": 0.00021966038912069052,
    "ml": 1.4948076568543911e-05,
    "cy": 0.006319893989712,
    "sk": 8.255112334154546e-06,
    "te": 1.980298475245945e-05,
    "fa": 0.00011575288226595148,
    "lv": 8.700824309926247e-07,
    "bn": 1.1641573109955061e-05,
    "sr": 9.02842458572195e-08,
    "az": 3.33536695507064e-06,
    "sl": 1.7205131371156313e-05,
    "kn": 3.46094878977965e-07,
    "et": 1.6255279433607939e-06,
    "mk": 6.452324186057012e-08,
    "br": 0.013589906506240368,
    "eu": 2.0114837752771564e-05,
    "is": 4.929352144245058e-06,
    "hy": 1.0660472753443173e-06,
    "ne": 2.0548598058667267e-06,
    "mn": 5.0568687584018335e-05,
    "bs": 3.441242597546079e-06,
    "kk": 4.957580586051336e-07,
    "sq": 1.9719890076430602e-07,
    "sw": 7.754960279271472e-06,
    "gl": 7.02076795278117e-05,
    "mr": 1.785293193279358e-06,
    "pa": 4.558905857265927e-06,
    "si": 6.0051803302485496e-05,
    "km": 0.001573145855218172,
    "sn": 0.0003915869747288525,
    "yo": 0.00011575288226595148,
    "so": 1.5969655464687094e-07,
    "af": 8.001128662726842e-06,
    "oc": 0.0006356084486469626,
    "ka": 1.0150930052077456e-07,
    "be": 1.0766695595521014e-05,
    "tg": 1.0071934752886591e-07,
    "sd": 1.220026479131775e-05,
    "gu": 1.4093170364049001e-07,
    "am": 8.56593089793023e-07,
    "yi": 1.3824710549670272e-05,
    "lo": 8.001128662726842e-06,
    "uz": 4.0063390827072e-08,
    "fo": 2.388695975241717e-05,
    "ht": 0.0004040172789245844,
    "ps": 3.2836567243066384e-06,
    "tk": 1.9719890076430602e-07,
    "nn": 0.07120601832866669,
    "mt": 1.1708239071595017e-06,
    "sa": 0.0002910027396865189,
    "lb": 6.671153300885635e-07,
    "my": 2.7926684197154827e-05,
    "bo": 7.516364803450415e-06,
    "tl": 7.243630534503609e-05,
    "mg": 5.61767421913828e-07,
    "as": 1.0660472753443173e-06,
    "tt": 2.5071027920375855e-08,
    "haw": 0.014466380700469017,
    "ln": 2.9727802029810846e-05,
    "ha": 3.40729144454599e-07,
    "ba": 2.0666269051616837e-07,
    "jw": 0.006833434104919434,
    "su": 3.46094878977965e-07
  }
}